src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  for (const value in robot) {
+    const isKey = robot[value];
+
+    if (!(`${isKey}` in robot)) {
```

Comment:
invert the condition — if there is such a key, return `null` immediately

Reply:
done